www-static/client/connect.js

```diff
@@ -194,17 +194,22 @@ CCC.init = function() {
   document.body.addEventListener('click', function() {
     CCC.postToAllFrames({'mode': 'blur'});
   }, true);
-  CCC.tab('world');
+  CCC.tab();
   CCC.schedulePing(0);
   // Firefox sometimes caches the disabled value on reload.
   CCC.commandTextarea.disabled = false;
 };
 
 /**
  * Switch between world and log views.
- * @param {string} mode Either 'world' or 'log'.
+ * @param {string=} mode Either 'world' or 'log', or undefined.
  */
 CCC.tab = function(mode) {
+  if (!mode) {
+    // Check for a cookie preference.
+    var m = document.cookie.match(/(?:^|;\s*)tab=(\w+)(?:;|$)/);
```

Comment:
Ugh—is that really how one is supposed to find a cookie value from JS?  How wretched. 
 On the other hand, cute regexp: `?:^|`.